tools/panda.py

```diff
@@ -0,0 +1,140 @@
+#!/usr/bin/python3
```

Comment:
Do we need license headers for tools we don't ship?

Reply:
probably not, but i added one anyway